keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -292,29 +546,245 @@ N/A
 
 N/A
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+It's up to the user what package repository (OBS or Google) they want to use.
+In case OBS doesn't work for them, they can reconfigure their systems to use
+the Google package repository.
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+N/A. This is configured on the operating system (i.e. package manager) level. 
+
+###### Does enabling the feature change any default behavior?
+
+Not anticipated. We're trying to match the existing spec files as best as we
+can.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. Users can rollback to the Google package repository.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+There are no side effects anticipated.
+
+###### Are there any tests for feature enablement/disablement?
+
+N/A
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+N/A
+
+###### What specific metrics should inform a rollback?
+
+Installation and upgrading issues. For example, if a package upgrade is not
+possible due to some error.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+We'll ask openSUSE to provide us with metrics on the repository usage. We don't
+have any metrics for the Google repository and there's no way that we can
+get those metrics.
```

Comment:
> We don't have any metrics for the Google repository and there's no way that we can get those metrics.

👀

Reply:
We already checked with Ben Elder and Google Build Admins, and there's no any way to get those metrics.